trie/trie.go

```diff
@@ -355,7 +468,8 @@ func (t *Trie) insert(n node, prefix, key []byte, value node) (bool, node, error
 		if err != nil {
```

Comment:
where init trie node's shadow nodes?

Reply:
ChildExpired function will call function GetShadownode which can load shadow node from cache or disk.

Reply:
may could use trie.sndb to fetch shadow nodes? full node cannot handle shadow nodes query